sdk/iot/core/inc/az_iot_core.h

```diff
@@ -122,6 +122,15 @@ AZ_NODISCARD int32_t az_iot_retry_calc_delay(
  */
 AZ_NODISCARD az_span az_span_token(az_span source, az_span delimiter, az_span* out_remainder);
 
+/**
+ * @brief utoa_size gives the length, in bytes, of the string that would represent the given number.
```

Comment:
Remove the method name (which is also out of sync atm)
```suggestion
 * @brief Gives the length, in bytes, of the string that would represent the given number.
```